components/gitpod-cli/cmd/top.go

```diff
@@ -57,23 +127,8 @@ func getColor(severity api.ResourceStatusSeverity) int {
 	}
 }
 
-var topCmd = &cobra.Command{
-	Use:   "top",
-	Short: "Display usage of workspace resources (CPU and memory)",
-	Run: func(cmd *cobra.Command, args []string) {
-		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
-		defer cancel()
-
-		workspaceResources, err := supervisor_helper.GetWorkspaceResources(ctx)
-		if err != nil {
-			log.Fatalf("cannot get workspace resources: %s", err)
-		}
-
-		outputTable(workspaceResources)
-	},
-}
-
 func init() {
 	topCmd.Flags().BoolVarP(&noColor, "no-color", "", false, "Disable output colorization")
+	topCmd.Flags().BoolVarP(&opts.Json, "json", "j", false, "print like json")
```

Comment:
```suggestion
	topCmd.Flags().BoolVarP(&opts.Json, "json", "j", false, "output in json format")
```

Reply:
I would suggest `output in json format` or `output data as json`

Reply:
I would use uppercase JSON, since it is an abbreviation [[1](https://www.json.org/json-en.html)]

```suggestion
	topCmd.Flags().BoolVarP(&opts.Json, "json", "j", false, "output in JSON format")
```

Or even make it super-short

```suggestion
	topCmd.Flags().BoolVarP(&opts.Json, "json", "j", false, "output JSON")
```